.github/dockerursim/build_and_run_docker_ursim.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+
+docker build ${DIR} -t mydockerursim
+docker volume create dockerursim
+docker run --name="mydockerursim" -d \
+  -e ROBOT_MODEL=UR5 \
+  -p 8080:8080 \
+  -p 29999:29999 \
+  -p 30001-30004:30001-30004 \
+  -v "${DIR}/.vol":/ursim/programs \
+  -v dockursim:/ursim \
+  --privileged \
```

Comment:
This is a bit unfortunate, but apparently `URControl` cannot be started without it.

Reply:
I agree with both points.